wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -1082,6 +1082,7 @@ public final class SessionManager: NSObject, SessionManagerType {
         deleteTemporaryData()
 
         PrivateUserDefaults.removeAll(forUserID: account.userIdentifier, in: sharedUserDefaults)
+        PrivateUserDefaults.removeAll(forUserID: account.userIdentifier, in: .standard)
```

Comment:
Why is this change needed?

Reply:
I remove the user default storage which includes the notification badge on logout.